packages/web/src/components/nav/desktop/NowPlayingArtworkTile.tsx

```diff
@@ -9,9 +9,9 @@ import {
   averageColorSelectors
 } from '@audius/common'
 import { IconButton } from '@audius/stems'
+import { animated, useSpring } from '@react-spring/web'
```

Comment:
I know the APIs changed considerably between versions. I'm assuming you audited the usage of `useSpring` here to make sure we are configuring things correctly based on the current docs with respect to what the old docs say?
(I think you have to use internet archive to get a reliable version of the old docs 😬 )

Reply:
yeah exactly, it was the one component i knew before/after, and referencing the docs and its the same api for this one, since its config object.